core/server/common/src/main/java/alluxio/master/journal/DelegatingJournaled.java

```diff
@@ -11,11 +11,12 @@
 
 package alluxio.master.journal;
 
+import alluxio.master.journal.checkpoint.CheckpointInputStream;
+import alluxio.master.journal.checkpoint.CheckpointName;
```

Comment:
Is this import necessary?

Reply:
we moved the class from `journal` to `journal.checkpoint`